tests/test_networks.py

```diff
@@ -321,21 +328,35 @@ def test_redis_dsns():
     class Model(BaseModel):
         a: RedisDsn
 
-    m = Model(a='redis://user:pass@localhost:5432/app')
-    assert m.a == 'redis://user:pass@localhost:5432/app'
+    m = Model(a='redis://user:pass@localhost:6379/app')
```

Comment:
I reckon it's better to keep `5432` to first try with a different port than the default one

Reply:
Perhaps 5432 is a bad choice as an "non-default" port as it's quite well known as the postgres port and people might be confused (or think it's a typo).

Reply:
@daviskirk  I agree. A dynamic port (49152-65535) may be a better choice.